substrait_consumer/tests/functional/relations/project_relation_snapshots/DataFusionProducer/project_all_col_plan.json

```diff
@@ -0,0 +1,46 @@
+{
+  "relations": [
+    {
+      "root": {
+        "input": {
+          "read": {
+            "baseSchema": {
+              "names": [
+                "r_regionkey",
+                "r_name",
+                "r_comment"
+              ]
+            },
+            "projection": {
```

Comment:
Technically this isn't a project but a narrowing of the returned results of the read.  It's correct and probably faster but it's not testing the project relation behavior.